### ISSUE TITLE
Support rkt as a local container engine

### ISSUE THREAD

**Issue by j00bar** [ansible/ansible-container#225]

**Comment by srizzling** [comment]
Whats required for something like this to kickstart? 0.4 milestone is pretty close :P

**Comment by TomasTomecek** [comment]
@srizzling I think that biggest issue will be that there is no rkt-compose, so the orchestration for rkt needs to be done from scratch.

**Comment by srizzling** [comment]
Yeah, I was kind of implying that is the milestone 0.4 is very tight for a big task like this. 

rkt also doesn't support the full lifecycle that docker supports either (commit, push and pull).
